tests/Http/TagsTest.php

```diff
@@ -56,10 +58,11 @@ public function testTaggingAPostWithoutActivityScope()
 
         // Apply the tag to the post
         $response = $this->postJson('api/v3/posts/' . $post->id . '/tags', [
-            'tag_name' => 'Good Submission',
+            'tag_name' => 'Group Photo',
```

Comment:
Changed a number of these tag names to be `Group Photo` since any tag with `Good *` will trigger a Slack notification. Most of our devs don't see this happen locally since they don't have the `SLACK_WEBHOOK_INTEGRATION_URL` env variable set, but I did and was seeing notifications go through on the test channel when running `phpunit`.

Reply:
Left a comment [below](#discussion_r462356992)... I think we can keep all these as-is by swapping out the Slack webhook URL when running tests!